src/components/TodoComponents/TodoForm.js

```diff
@@ -4,15 +4,19 @@ import './Todo.css';
 const TodoForm = props => {
     return (
         <form>
-            <input
-                onChange={props.handleToDoChange}
-                type="text"
-                name="todo"
-                value={props.value}
-                placeholder="**ToDo**"
-            />
-            <button onClick={props.handleAddTodo}>Add ToDo</button>
-            <button onClick={props.handleClearTodos}>Clear Completed</button>
+            <div className="inputContainer"> 
+                <input className="input"
+                    onChange={props.handleToDoChange}
+                    type="text"
+                    name="todo"
+                    value={props.value}
+                    placeholder="**ToDo**"
```

Comment:
You can also add a `required` to tell the user they can't submit an empty todo :)